### PR TITLE
OSDOCS-11423#removing redundant prerequisites

### PR DESCRIPTION
**Pull request by brendan-daly-red-hat**
Versions: 
4.17+

Issue:
https://issues.redhat.com/browse/OSDOCS-11423

Link to docs preview:

**Azure**
- https://80860--ocpdocs-pr.netlify.app/openshift-enterprise/latest/installing/installing_azure_stack_hub/upi/installing-azure-stack-hub-user-infra#installation-azure-create-resource-group-and-identity_installing-azure-stack-hub-user-infra
- https://80860--ocpdocs-pr.netlify.app/openshift-enterprise/latest/installing/installing_azure_stack_hub/upi/installing-azure-stack-hub-user-infra#installation-azure-user-infra-uploading-rhcos_installing-azure-stack-hub-user-infra
- https://80860--ocpdocs-pr.netlify.app/openshift-enterprise/latest/installing/installing_azure_stack_hub/upi/installing-azure-stack-hub-user-infra#installation-azure-create-dns-zones_installing-azure-stack-hub-user-infra
- https://80860--ocpdocs-pr.netlify.app/openshift-enterprise/latest/installing/installing_azure_stack_hub/upi/installing-azure-stack-hub-user-infra#installation-creating-azure-vnet_installing-azure-stack-hub-user-infra 
- https://80860--ocpdocs-pr.netlify.app/openshift-enterprise/latest/installing/installing_azure/upi/installing-azure-user-infra#installation-azure-user-infra-deploying-rhcos_installing-azure-user-infra 
-  https://80860--ocpdocs-pr.netlify.app/openshift-enterprise/latest/installing/installing_azure/upi/installing-azure-user-infra#installation-creating-azure-dns_installing-azure-user-infra 
- https://80860--ocpdocs-pr.netlify.app/openshift-enterprise/latest/installing/installing_azure/upi/installing-azure-user-infra#installation-creating-azure-bootstrap_installing-azure-user-infra
- https://80860--ocpdocs-pr.netlify.app/openshift-enterprise/latest/installing/installing_azure/upi/installing-azure-user-infra#installation-azure-user-infra-wait-for-bootstrap_installing-azure-user-infra
- https://80860--ocpdocs-pr.netlify.app/openshift-enterprise/latest/installing/installing_azure/upi/installing-azure-user-infra#installation-creating-azure-worker_installing-azure-user-infra
- https://80860--ocpdocs-pr.netlify.app/openshift-enterprise/latest/installing/installing_azure/upi/installing-azure-user-infra#installation-creating-azure-control-plane_installing-azure-user-infra 

**Azure Stack Hub**
- https://80860--ocpdocs-pr.netlify.app/openshift-enterprise/latest/installing/installing_azure_stack_hub/upi/installing-azure-stack-hub-user-infra#installation-azure-create-resource-group-and-identity_installing-azure-stack-hub-user-infra 
- https://80860--ocpdocs-pr.netlify.app/openshift-enterprise/latest/installing/installing_azure_stack_hub/upi/installing-azure-stack-hub-user-infra#installation-azure-user-infra-uploading-rhcos_installing-azure-stack-hub-user-infra
- https://80860--ocpdocs-pr.netlify.app/openshift-enterprise/latest/installing/installing_azure_stack_hub/upi/installing-azure-stack-hub-user-infra#installation-azure-create-dns-zones_installing-azure-stack-hub-user-infra
- https://80860--ocpdocs-pr.netlify.app/openshift-enterprise/latest/installing/installing_azure_stack_hub/upi/installing-azure-stack-hub-user-infra#installation-creating-azure-vnet_installing-azure-stack-hub-user-infra 
- https://80860--ocpdocs-pr.netlify.app/openshift-enterprise/latest/installing/installing_azure_stack_hub/upi/installing-azure-stack-hub-user-infra#installation-azure-user-infra-deploying-rhcos_installing-azure-stack-hub-user-infra
- https://80860--ocpdocs-pr.netlify.app/openshift-enterprise/latest/installing/installing_azure_stack_hub/upi/installing-azure-stack-hub-user-infra#installation-creating-azure-dns_installing-azure-stack-hub-user-infra
- https://80860--ocpdocs-pr.netlify.app/openshift-enterprise/latest/installing/installing_azure_stack_hub/upi/installing-azure-stack-hub-user-infra#installation-creating-azure-bootstrap_installing-azure-stack-hub-user-infra
- https://80860--ocpdocs-pr.netlify.app/openshift-enterprise/latest/installing/installing_azure_stack_hub/upi/installing-azure-stack-hub-user-infra#installation-azure-user-infra-wait-for-bootstrap_installing-azure-stack-hub-user-infra 
- https://80860--ocpdocs-pr.netlify.app/openshift-enterprise/latest/installing/installing_azure_stack_hub/upi/installing-azure-stack-hub-user-infra#installation-creating-azure-worker_installing-azure-stack-hub-user-infra
- https://80860--ocpdocs-pr.netlify.app/openshift-enterprise/latest/installing/installing_azure_stack_hub/upi/installing-azure-stack-hub-user-infra#installation-creating-azure-control-plane_installing-azure-stack-hub-user-infra

QE review:
- [x] QE has approved this change.

Additional information:

### CODE REVIEW

File: modules/installation-azure-user-infra-wait-for-bootstrap.adoc [hunk 1]
```diff
@@ -27,12 +27,6 @@ the Ignition config files that you generated with the installation program.
 
 .Prerequisites
 
-* Configure an Azure account.
-* Generate the Ignition config files for your cluster.
-* Create and configure a VNet and associated subnets in {cp}.
-* Create and configure networking and load balancers in {cp}.
-* Create control plane and compute roles.
-* Create the bootstrap machine.
 * Create the control plane machines.
```
Comment (bscott-rh) [inline modules/installation-azure-user-infra-wait-for-bootstrap.adoc:30]: Can the whole thing be removed?

File: modules/installation-creating-azure-bootstrap.adoc [hunk 2]
```diff
@@ -35,9 +35,6 @@ support with your installation logs.
 
 .Prerequisites
 
-* Configure an Azure account.
-* Generate the Ignition config files for your cluster.
-* Create and configure a VNet and associated subnets in {cp}.
 * Create and configure networking and load balancers in {cp}.
 * Create control plane and compute roles.
```
Comment (bscott-rh) [inline modules/installation-creating-azure-bootstrap.adoc:39]: Thinking this whole block can go too

File: modules/installation-creating-azure-control-plane.adoc [hunk 3]
```diff
@@ -36,11 +36,6 @@ If you do not use the provided ARM template to create your control plane machine
 
 .Prerequisites
 
-* Configure an Azure account.
-* Generate the Ignition config files for your cluster.
-* Create and configure a VNet and associated subnets in {cp}.
-* Create and configure networking and load balancers in {cp}.
-* Create control plane and compute roles.
 * Create the bootstrap machine.
```
Comment (bscott-rh) [inline modules/installation-creating-azure-control-plane.adoc:39]: Same comment as above

File: modules/installation-azure-user-infra-wait-for-bootstrap.adoc (hunk 1 above)
Comment (brendan-daly-red-hat) [reply modules/installation-azure-user-infra-wait-for-bootstrap.adoc:30]: This is the first instance of ' Create the control plane machines.' in the assembly, so I think it needs to stay.

File: modules/installation-creating-azure-bootstrap.adoc (hunk 2 above)
Comment (brendan-daly-red-hat) [reply modules/installation-creating-azure-bootstrap.adoc:39]: This is the first instance of 'Create and configure networking and load balancers' and 'Create control plane and compute roles' in the assembly, so I left them in here and removed them from later modules.

File: modules/installation-creating-azure-control-plane.adoc (hunk 3 above)
Comment (brendan-daly-red-hat) [reply modules/installation-creating-azure-control-plane.adoc:39]: This is the first instance of 'Create the bootstrap machine' in the assembly, so I left it here and removed it from later modules, (e.g., [Wait for bootstrap completion and remove bootstrap resources](https://80860--ocpdocs-pr.netlify.app/openshift-enterprise/latest/installing/installing_azure/upi/installing-azure-user-infra#installation-azure-user-infra-wait-for-bootstrap_installing-azure-user-infra) module).

**Review by xenolinux — COMMENTED** [review]
LGTM

**Review by xenolinux — COMMENTED** [review]
Merge review LGTM

**Comment by xenolinux** [comment]
Reason of conflict in the cherry-pick commands below 4.17: HCP content migration is done from 4.17+ only.

**Pull request merged**